[2023-09-11 11:52:57,241] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_18.extract_gz scheduled__2023-09-10T05:00:00+00:00 [queued]>
[2023-09-11 11:52:57,257] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: listing_4_18.extract_gz scheduled__2023-09-10T05:00:00+00:00 [queued]>
[2023-09-11 11:52:57,257] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:52:57,257] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-11 11:52:57,258] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-11 11:52:57,288] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): extract_gz> on 2023-09-10 05:00:00+00:00
[2023-09-11 11:52:57,295] {standard_task_runner.py:52} INFO - Started process 619 to run task
[2023-09-11 11:52:57,298] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'listing_4_18', 'extract_gz', 'scheduled__2023-09-10T05:00:00+00:00', '--job-id', '362', '--raw', '--subdir', 'DAGS_FOLDER/listing_4_18.py', '--cfg-path', '/tmp/tmpednhxvay', '--error-file', '/tmp/tmp0e8bjax5']
[2023-09-11 11:52:57,299] {standard_task_runner.py:80} INFO - Job 362: Subtask extract_gz
[2023-09-11 11:52:57,375] {task_command.py:371} INFO - Running <TaskInstance: listing_4_18.extract_gz scheduled__2023-09-10T05:00:00+00:00 [running]> on host b18943745c0d
[2023-09-11 11:52:57,462] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=listing_4_18
AIRFLOW_CTX_TASK_ID=extract_gz
AIRFLOW_CTX_EXECUTION_DATE=2023-09-10T05:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-10T05:00:00+00:00
[2023-09-11 11:52:57,463] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-09-11 11:52:57,464] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip --force /tmp/wikipageviews.gz']
[2023-09-11 11:52:57,480] {subprocess.py:85} INFO - Output:
[2023-09-11 11:52:58,315] {subprocess.py:92} INFO - 
[2023-09-11 11:52:58,315] {subprocess.py:92} INFO - gzip: /tmp/wikipageviews.gz: invalid compressed data--format violated
[2023-09-11 11:52:58,328] {subprocess.py:96} INFO - Command exited with return code 1
[2023-09-11 11:52:58,342] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-09-11 11:52:58,345] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=listing_4_18, task_id=extract_gz, execution_date=20230910T050000, start_date=20230911T115257, end_date=20230911T115258
[2023-09-11 11:52:58,360] {standard_task_runner.py:97} ERROR - Failed to execute job 362 for task extract_gz (Bash command failed. The command returned a non-zero exit code 1.; 619)
[2023-09-11 11:52:58,394] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-09-11 11:52:58,438] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
